research/fedbn/create_job.sh

```diff
@@ -0,0 +1 @@
+nvflare job create -force -j ./jobs/client_api -w sag_pt -sd ./src/ -f config_fed_client.conf app_script=fedbn_cifar10.py
```

Comment:
since you already provide this create job script here, we don't need to check-in the created job in the repo.

Reply:
suggest using 
`nvflare job create -force -j ./jobs/fed_bn -w sag_pt -sd ./src/ -f config_fed_client.conf app_script=fedbn_cifar10.py`